README.md

```diff
@@ -1,8 +1,8 @@
 <!-- PROJECT SHIELDS -->
 [![Forks][forks-shield]][forks-url]
 [![Stargazers][stars-shield]][stars-url]
-[![Issues-open][issues-open-shield]][issues-url]
-[![Issues-closed][issues-closed-shield]][issues-url]
+[![Issues-open][issues-open-shield]](https://github.com/Israel-Laguan/microverse-trial-3/issues?q=is%3Aopen+is%3Aissue)
+[![Issues-closed][issues-closed-shield]](https://github.com/Israel-Laguan/microverse-trial-3/issues?q=is%3Aissue+is%3Aclosed)
```

Comment:
Please move these URL to the bottom, please see that you can set a "variable" like this:

```markdown
[issues-url-open]: https://github.com/Israel-Laguan/microverse-trial-3/issues?q=is%3Aopen+is%3Aissue
[issues-url-closed]: https://github.com/Israel-Laguan/microverse-trial-3/issues?q=is%3Aissue+is%3Aclosed
```

Reply:
solved, thanks

Reply:
Thanks to you!